src/amo/components/GuidesAddonCard/index.js

```diff
@@ -121,7 +121,7 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
               <div className="GuidesAddonCard-content-header">
                 <div className="GuidesAddonCard-content-header-title">
                   <span className="GuidesAddonCard-content-header-authors">
-                    <AddonTitle addon={addon} as="span" />
+                    <AddonTitle addon={addon} as="span" linkToAddon />
```

Comment:
ah, that's cool!

Reply:
yes all made very easy by the AddonTitle component:)